### ISSUE TITLE
Pupil plane orientation for Roman Space Telescope

### ISSUE THREAD

**Issue by rmjarvis** [spacetelescope/webbpsf#374]
I have a question about the pupil plane files hosted here: https://roman.gsfc.nasa.gov/science/Roman_Reference_Information.html

Basically, which way is up?

Are they oriented so that up is the Observatory FOV +Z direction?  (Towards the sunshade.)  
Or is up is the +y direction on the WFC SCAs, which is 60 degrees rotated from the observatory FOV direction?
Or something else entirely? (Although I strongly suspect it is one of those first two.)

In other words, if I make a PSF the normal way by applying these images as the pupil mask, and then draw it in image coordinates on an SCA, will the diffraction spikes be pointing the correct directions?  Or will I need to rotate by 60 degrees?  The Reference Information page doesn't specify this anywhere that I could find.

Thanks!

**Comment by Skyhawk172** [comment]
I checked with our colleagues at Goddard and, based on some Code V and Zemax modeling, they confirmed that the pupil FITS files available online are in the focal plane array coordinate system. So that would correspond to your second option. There should be no rotation or flip needed. I will look into it some more, but I hope that helps for now.

**Comment by mperrin** [comment]
Thanks for checking this Charles. 

Should we perhaps add a note with this information to the Roman docs page at https://webbpsf.readthedocs.io/en/latest/wfirst.html?

**Comment by rmjarvis** [comment]
Great!  Thanks very much for the info.
